### PR TITLE
Add compatibility for packing with webpack

### PR DESCRIPTION
**Pull request by HerrZatacke**
This should fix issue https://github.com/thenickdude/webm-writer-js/issues/16

I ran your tests `npm run test` as well which did work for me, but these only cover the electron-side as far as I can see, so please check if the "plain" browser version works for you as well.

### CODE REVIEW

File: src/BlobBuffer.js [hunk 1]
```diff
@@ -220,7 +220,7 @@
 	};
 	
 	if (typeof module !== 'undefined' && typeof module.exports !== 'undefined') {
-		module.exports = BlobBuffer(require('fs'));
+		module.exports = BlobBuffer;
 	} else {
 		window.BlobBuffer = BlobBuffer(null);
```
Comment (Cobertos) [inline src/BlobBuffer.js:225]: It feels a little odd to me to have one platform exporting a function and the other exporting the return of the function. Maybe change `main.js` to call the function like `browser.js` does?

As a future features, I've also [seen a pattern in other libraries to pass in fs as an external](https://isomorphic-git.org/docs/en/fs). It'd be nice if the user could just pass in whatever implementation of `fs` for whatever their needs are, meaning you could use anything from [`BrowserFS`](https://github.com/jvilk/BrowserFS)

**Comment by chrisgervang** [comment]
Any chance this (or something that solves the same issue) can be merged in? I've also encountered the issue with next.js https://github.com/visgl/hubble.gl/issues/158

**Comment by ilijapuaca** [comment]
+1 for landing this in some capacity, we're stuck with the same issue when trying to make use of [esbuild](https://esbuild.github.io/)

**Comment by chrisgervang** [comment]
I linked this version of the library to the hubble.gl repo locally and ran the browser through some render tests. Great news! No runtime errors, and I was able to remove my webpack workaround. This seems like a great PR to merge and publish - much appreciated for putting it together @HerrZatacke, and thanks for your hard work @thenickdude!

**Pull request merged**